scania-digital-design-system/tegel
# fix(table): passing on unit value with horizontalscroll

## **Describe pull-request**  
Fixing broken horizontal scroll

## **Issue Linking:**  
- **Jira:** [CDEP-3450](https://tegel.atlassian.net/browse/CDEP-3450)    

## **How to test**  
_Provide detailed steps for testing, including any necessary setup._
1. Go to Storybook.
2. Check in Storybook
4. Enter a value for horizontal scroll


[CDEP-3450]: https://tegel.atlassian.net/browse/CDEP-3450?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
